main/webapp/modules/core/langs/translation-en.json

```diff
@@ -284,8 +284,9 @@
     "core-dialogs/for-include-if-exist-drop-stmt-checkbox": "Include 'IF EXISTS' in DROP statement",
     "core-dialogs/for-null-cell-value-to-empty-str-label": "Convert null value to NULL in INSERT",
     "core-dialogs/choose-export-destination": "Please choose the destination for project export",
-    "core-dialogs/export-to-local": "Export to local",
-    "core-dialogs/export-to-google-drive": "Export to Google Drive",
+    "core-dialogs/export-to-local": "OpenRefine project archive to file",
+    "core-dialogs/export-to-google-drive": "OpenRefine project archive to Google Drive...",
+    "core-dialogs/export-to-google-sheets": "OpenRefine project archive to Google Sheets...",
```

Comment:
The Google Sheets option should be in the second section and should not be named "OpenRefine project archive to Google Sheets", since it does not export the project archive, it just exports the current state of the table like the other exporters. I think we could just use "Google Sheets" as label for that item.

Also these translations should be in the `gdata` extension (since the menu items should be added in the `gdata` extension, see below).